generate-docs/api-extractor-inputs-custom-functions-runtime/custom-functions-runtime.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Custom Functions 1.4
+// Type definitions for non-npm package Custom Functions 1.4
```

Comment:
Something similar to "Type definitions for non-npm package Custom Functions 1.4" appears several places in this PR. Can this be clarified? Does it mean definitions for CF that are not in any package at all? Or does it mean they are in a package, but not an npm package? Or does it mean some third thing?

Reply:
This change was made by DefinitelyTyped folks.